cloudformation_seed/cfn_stackset.py

```
@@ -437,6 +437,7 @@ def deploy(self) -> None:
             self.update_stackset()
         self.stack = self.find_existing_stackset()
         self.rollout_stackset()
+        self.check_stack_instances()
```

Hey Stan, yes! We need this. The silent failures are annoying.